docs/components/alerts/snippets/alerts.hbs

```diff
@@ -0,0 +1,19 @@
+---
+title: Alerts
+layout: demo.hbs
+---
+<div class="alert alert-success" role="alert">
+  <strong>Well done!</strong> You successfully read this important alert message.
+</div>
+<div class="alert alert-info" role="alert">
+  <strong>Heads up!</strong> This alert needs your attention, but it's not super important.
+</div>
+<div class="alert alert-warning alert-dismissible" role="alert">
+  <strong>Warning!</strong> Pushing the x on the right will remove this alert. For EVER!
+  <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+    <span aria-hidden="true">&times;</span>
+  </button>
+</div>
+<div class="alert alert-danger" role="alert">
+  <strong>Danger!</strong> Are you sure you want to release the raptors?
+</div>
```

Comment:
Wondering if this could also be split up into multiple snippets, so it's easier to figure out which example belongs to what code snippet. Thoughts @elessar-ch @FKSI?